src/lib/pageVideoStreamWriter.ts

```diff
@@ -59,6 +59,14 @@ export default class PageVideoStreamWriter extends EventEmitter {
     return width !== null && height !== null ? `${width}x${height}` : '100%';
   }
 
+  private get autopad(): { activation: boolean; color?: string } {
+    const autopad = this.options.autopad;
+
+    return !autopad
```

Comment:
Sorry I forgot to mention one more stuff. 

For all configuration change, I would recommend to add a test case and could you please ensure the test pass. 

Please let me know if you find any difficulty. I can help you over this. 


Thanks.

Reply:
@prasanaworld never mind :)

I founded the test code you added 2 days ago. I wrote it according to the area, so please check it.

with my best regards.